data/corsika/corsika_parameters.yml

```diff
@@ -0,0 +1,105 @@
+USER_PARAMETERS:
+  RUNNR:
+    len: 1
+    names: ['RUNNUMBER', 'NRUN']
+    default: 1
+  EVTNR:
+    len: 1
+    names: ['EVENTNR', 'EVENTNUMBER']
+    default: 1
+  NSHOW:
+    len: 1
+    names: ['NSHOWERS']
+  PRMPAR:
+    len: 1
+    names: ['PRIMARY']
+  ERANGE:
+    len: 2
+    names: ['ENRANGE', 'ENERGYRANGE']
+    unit: [!astropy.units.Unit {unit: GeV}, !astropy.units.Unit {unit: GeV}]
+  ESLOPE:
+    len: 1
+    names: ['ENSLOPE', 'ENERGYSLOPE']
+  THETAP:
+    len: 2
+    names: ['THETA', 'ZENITH']
+    unit: [!astropy.units.Unit {unit: deg}, !astropy.units.Unit {unit: deg}]
+  PHIP:
+    len: 2
+    names: ['PHI', 'AZIMUTH']
+    unit: [!astropy.units.Unit {unit: deg}, !astropy.units.Unit {unit: deg}]
+  VIEWCONE:
+    len: 2
+    names: ['CONE']
+    unit: [!astropy.units.Unit {unit: deg}, !astropy.units.Unit {unit: deg}]
+  CSCAT:
+    len: 3
+    names: ['SCAT']
+    unit: [null, !astropy.units.Unit {unit: cm}, null]
+
+PRIMARIES:
+  GAMMA:
+    number: 1
+    names: ['PHOTON']
+  ELECTRON:
+    number: 3
+    names: ['ELECTRONS']
+  PROTON:
+    number: 14
+    names: ['PROTONS']
+  HELIUM:
+    number: 402
+    names: []
+
+SITE_PARAMETERS:
+  South:
+    OBSLEV: [2150.e2]
+    ATMOSPHERE: [26, 'Y']
+    MAGNET: [20.925, -9.119]
+    ARRANG: [-3.433]
+  North:
+    OBSLEV: [2158.e2]
+    ATMOSPHERE: [36, 'Y']
+    MAGNET: [30.576, 23.571]
+    ARRANG: [-5.3195]
+
+INTERACTION_FLAGS:
+  FIXHEI: [0., 0]
+  FIXCHI: [0.]
+  TSTART: ['T']
+  ECUTS: [0.3, 0.1, 0.020, 0.020]
+  MUADDI: ['F']
+  MUMULT: ['T']
+  LONGI: ['T', 20., 'F', 'F']
+  MAXPRT: [0]
+  ECTMAP: [1.e6]
+#  STEPFC: [1.0]
+
+CHERENKOV_EMISSION_PARAMETERS:
+  CERSIZ: [5.]
+  CERFIL: [0]
+  CWAVLG: [240., 700.]
+
+DEBUGGING_OUTPUT_PARAMETERS:
+  DEBUG: ['F', 6, 'F', 1000000]
+  DATBAS: ['yes']
+  DIRECT: ['/dev/null']
+
+IACT_TUNING_PARAMETERS:
+  IACT: [
+    ['SPLIT_AUTO', '15M'],
+    ['IO_BUFFER', '800MB'],
+    ['MAX_BUNCHES', '1000000']
+  ]
+
+# Telescopes info
+TELESCOPE_Z: # Z coordinate in m
```

Comment:
Are you sure you want to use units of 'm' here? The CORSIKA input is in 'cm'

Reply:
This info is actually being carried by the layout files. Both TELESCOPE_Z and and TELESCOPE_SPHERE_RADIUS are there. We should discuss where we want to keep this, in the corsika_parameters or in each of the layout files.